heartbeat/rabbitmq-cluster

```diff
@@ -181,26 +181,16 @@ remove_pid () {
 rmq_monitor() {
 	local rc
 
-	$RMQ_CTL cluster_status > /dev/null 2>&1
-	rc=$?
-	case "$rc" in
-	0)
+	if $RMQ_CTL eval 'rabbit_mnesia:cluster_status_from_mnesia().' | grep -q '^{ok'; then
```

Comment:
This should be fine I think. Another alternative seems to be to call 'rabbit_misc:which_applications().' and then check for the rabbit app, but from my small understanding this seems more solid.